ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/Claims.scala

```diff
@@ -118,6 +121,14 @@ object ClaimSet {
     def isAdmin: Either[AuthorizationError, Unit] =
       Either.cond(claims.contains(ClaimAdmin), (), AuthorizationError.MissingAdminClaim)
 
+    /** Returns true if the set of claims authorizes the user to use admin services or Identity Provider admin services, unless the claims expired */
+    def isAdminOrIDPAdmin: Either[AuthorizationError, Unit] =
+      Either.cond(
+        claims.contains(ClaimIdentityProviderAdmin) || claims.contains(ClaimAdmin),
```

Comment:
Idea: You could represent both `ClaimIdentityProviderAdmin` and `ClaimAdmin` as:
```
case class ClaimAdminOver(canManageIdp: IdpRange)

trait IdpRange
case object All extends IdpRange
case class Idp(idpId) extends IdpRange
```

This might be beneficial if you verifying idps is needed only for Admins/IdpAdmins